scene/gui/subviewport_container.cpp

```diff
@@ -190,6 +190,14 @@ void SubViewportContainer::_propagate_nonpositional_event(const Ref<InputEvent>
 		return;
 	}
 
+	if (GDVIRTUAL_IS_OVERRIDDEN(_propagate_input_event)) {
+		bool send = true;
+		GDVIRTUAL_CALL(_propagate_input_event, p_event, send);
+		if (!send) {
+			return;
+		}
+	}
```

Comment:
```suggestion
	bool send;
	if (GDVIRTUAL_CALL(_propagate_input_event, p_event, send)) {
		if (!send) {
			return;
		}
	}
```
`GDVIRTUAL_CALL()` returns false if the method is not overriden.

Reply:
Should still init `send` regardless, to be safe.

Reply:
If `send` is used only inside the condition it's guaranteed to be initialized.
I checked other `GDVIRTUAL_CALL` conditions and they are also uninitialized if not used afterwards.

Reply:
Sounds good, I will change that. I wasn't aware of that functionality. It is a bit difficult to guess the return value of this kind of macro.